lib/ur/usage_report.go

```diff
@@ -66,7 +72,8 @@ func New(cfg config.Wrapper, m model.Model, connectionsService connections.Servi
 // ReportData returns the data to be sent in a usage report with the currently
```

Comment:
Note that this comment was a lie, unfortunately.

Reply:
Why?